fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -408,7 +420,7 @@ private void createTombstone(final Node parent, final String path) throws Reposi
     public Date getCreatedDate() {
         try {
             if (hasProperty(JCR_CREATED)) {
-                return new Date(getProperty(JCR_CREATED).getDate().getTimeInMillis());
+                return new Date(getTimestamp(JCR_CREATED, 0L));
```

Comment:
The `0L` here matches up with this line: https://github.com/fcrepo4/fcrepo4/pull/1033/files/9f68918ebc5007adc656a619ff7916320ab56be6#diff-7aee50d5030524028026aa94cd75f350R465

The intention is to be able to use the `getTimestamp` method for both created and modified dates.  For modified dates, it's important to make sure that the returned value isn't before the created value.  But for created dates, that doesn't apply.  We could just revert the change to this line if that's confusing.